src/Dot/stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { Meta, Story } from '@storybook/react';
+
+import { Dot } from '.';
+import { Props } from './props';
+
+const meta: Meta = {
+  title: 'Dot',
+  component: Dot,
+  argTypes: {
+    onClick: { control: '' }
+  }
+}
+
+export default meta;
+
+const Template: Story<Props> = ({ avatarUrl, username, style, visible }) => (
+  <Dot avatarUrl={avatarUrl} username={username} style={style} visible={visible} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  avatarUrl: 'https://chat-engine-assets.s3.amazonaws.com/tutorials/my-face-min.png',
+  username: 'Dylan',
+  style: { float: 'right', marginLeft: '4px' },
+  visible: true
+};
+
+export const NoAvatar = Template.bind({});
+NoAvatar.args = {
+  avatarUrl: undefined,
+  username: 'Dylan',
+  style: { float: 'right', marginLeft: '4px' },
+  visible: true
+};
+
+export const TheirMessage = Template.bind({});
```

Comment:
I don't see any "their" logic in these components? Is there a difference / is this example educational?

Reply:
The point was to show how they would use it but I guess we will really just be passing this to another component so kinda useless- I'll take out